src/contracts/pods/EigenPod.sol

```diff
@@ -317,8 +317,20 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
             (withdrawalProofs.length == withdrawalFields.length), "EigenPod.verifyAndProcessWithdrawals: inputs must be same length"
         );
 
+        uint256 totalAmountToSend;
+        int256 totalSharesDelta;
         for (uint256 i = 0; i < withdrawalFields.length; i++) {
-            _verifyAndProcessWithdrawal(oracleTimestamp, withdrawalProofs[i], validatorFieldsProofs[i], validatorFields[i], withdrawalFields[i]);
+            WithdrawalInfo memory withdrawalInfo = _verifyAndProcessWithdrawal(oracleTimestamp, withdrawalProofs[i], validatorFieldsProofs[i], validatorFields[i], withdrawalFields[i]);
+            totalAmountToSend += withdrawalInfo.amountToSend;
+            totalSharesDelta += withdrawalInfo.sharesDelta;
+        }
+        // send ETH to the `recipient` via the DelayedWithdrawalRouter, if applicable
+        if (totalAmountToSend != 0) {
+            _sendETH_AsDelayedWithdrawal(podOwner, totalAmountToSend);
+        }
+        //update podOwner's shares in the strategy manager
+        if (totalSharesDelta != 0){
```

Comment:
nit: spacing
also noticing that we are inputting the `oracleTimestamp` into `_verifyAndProcessWithdrawal` repeatedly -- can we not move that logic out of the loop here too?

Reply:
good call, fixed